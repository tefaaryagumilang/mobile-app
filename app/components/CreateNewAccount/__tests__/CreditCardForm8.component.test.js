import React from 'react';
import renderer from 'react-test-renderer';
import {Provider} from 'react-redux';
import {createStore} from 'redux';
import {reduxForm} from 'redux-form';
import CreditCardForm from '../CreditCardForm8.component';

const spy = jest.fn();
const store = createStore(() => ({}));
const DecoratedForm = reduxForm({
  form: 'testForm',
  onSubmit: spy
}
)(CreditCardForm);

describe('CreditCardForm Component: PaymentSelection', () => {
  it('renders correctly', () => {
    const formValues = {};
    const accounts = [];
    const tree = renderer.create(
      <Provider store={store}>
        <DecoratedForm formValues={formValues} accounts={accounts}/>
      </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
