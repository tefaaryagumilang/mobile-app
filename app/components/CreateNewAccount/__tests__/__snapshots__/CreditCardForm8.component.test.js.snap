// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreditCardForm Component: PaymentSelection renders correctly 1`] = `
<KeyboardAwareScrollView
  contentContainerStyle={
    Object {
      "backgroundColor": "#FFFFFF",
      "flexGrow": 1,
      "justifyContent": "space-between",
      "paddingBottom": -10,
    }
  }
  enableOnAndroid={true}
  extraScrollHeight={100}
  keyboardShouldPersistTaps="handled"
>
  <View
    style={
      Object {
        "justifyContent": "space-between",
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "height": 7,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "flex": 8,
            },
            Object {
              "backgroundColor": "#FA3F70",
              "flexDirection": "row",
            },
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "flex": 2,
            },
            Object {
              "backgroundColor": "#DBDBDB",
              "flexDirection": "row",
            },
          ]
        }
      />
    </View>
    <View
      style={
        Object {
          "paddingHorizontal": 20,
          "paddingVertical": 20,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "paddingHorizontal": 20,
        "paddingVertical": 20,
      }
    }
  >
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "backgroundColor": "#FA3F70",
              "borderRadius": 30,
              "flexDirection": "row",
              "height": 47,
              "justifyContent": "center",
              "paddingHorizontal": 15,
            },
            undefined,
            Object {},
          ]
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 16,
            }
          }
        >
          BERIKUTNYA
        </Text>
      </View>
    </View>
  </View>
</KeyboardAwareScrollView>
`;
